adagrams/game.py

```diff
@@ -1,11 +1,144 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+    tiles = [] #list that's ten letters
+    count = 0 #going to stop at 10
+    letters_dict = LETTER_POOL.copy()
+    while count != 10:
+        letter = random.choice(list(letters_dict.keys()))
+        if letters_dict[letter] > 0:
+            letters_dict[letter] -= 1
+            tiles.append(letter)
+            count += 1
+        else:
+            continue
+    return tiles
+        
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_dict = {}
+    word_dict = {}
+    word = word.upper()
+
+    for letter in letter_bank:
+        if letter not in letter_bank_dict:
+            letter_bank_dict[letter] = 1
+        else:
+            letter_bank_dict[letter] += 1
```

Comment:
Nice job building up your frequency dictionary for your available letters! Another approach is to use the dictionary method .get() because it allows you to pass the function a default value to the method. 

```python
for letter in letter_bank:
    letter_bank_dict[letter] = letter_bank_dict.get(letter, 0) + 1
```

So we loop over letter_bank with a for loop, then we set the key of the dict to `letter_bank_dict[letter] ` and the value is what `letter_bank_dict.get(letter, 0) + 1` evaluates to. This is a pretty common pattern for building up a frequency dictionary so I wanted to call it out so you're familiar with it.